### PR TITLE
Development

### PR DESCRIPTION
**Pull request by mcrd25**

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -1,7 +1,991 @@
...
+    justify-content: space-around;
+}
+
+.c-pointer {
+    cursor: pointer;
+}
+
+/**** START Navigation ****/
+
+ul {
```
Comment (heyapricot) [inline styles/main.css]: Careful with duplicate selectors. You first used this one on line 19

File: styles/main.css [hunk 2]
```diff
@@ -1,7 +1,991 @@
...
+        align-self: center;
+    }
+
+    /* End Navigation */
+
+    /* cover*/
+    #cover {
+        height: 60vh;
+        flex-direction: row;
+        flex-flow: column wrap;
```
Comment (heyapricot) [inline styles/main.css:832]: This can conflict with the previous line

**Review by heyapricot — APPROVED** [review]
# Excellent job! It's almost identical 🤩
Hello! 👋🏼 Congratulations building this project. It looks great! Very similar to the original, even when the width changes! I liked your implementation a lot!

Some annotations:
1. Your borders for these sections are darker than the original
screen shot 2019-02-12 at 18 11 22
2. Your icons are smaller
screen shot 2019-02-12 at 18 13 30

You are ready to move on. No need from my side for further review.	
Again, great job! 🏅
Tips:
For every HTML/CSS projects please check your files with the [W3 Validator](https://validator.w3.org) and the [CSS Linter](http://csslint.net/)

File: styles/main.css (hunk 2 above)
Comment (adriaanbd) [reply styles/main.css:832]: Could you please expand on this comment, we'd like to understand what you mean by it?

File: styles/main.css (hunk 1 above)
Comment (adriaanbd) [reply styles/main.css]: Thanks for catching this one, resolved on commit https://github.com/adriaanbd/techy-media-site/pull/14/commits/cae50dc45ad218c11cfa49f78c3497c62d235c4e

**Pull request merged**